Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1474,8 +1497,18 @@ PhysicalParticleContainer::AddPlasmaFlux (amrex::Real dt)
                 } else {
                     theta = 2._rt*MathConst::pi*r.y;
                 }
-                pos.x = xb*std::cos(theta);
-                pos.y = xb*std::sin(theta);
+                Real const cos_theta = std::cos(theta);
+                Real const sin_theta = std::sin(theta);
+                // Rotate the position
+                pos.x = xb*cos_theta;
+                pos.y = xb*sin_theta;
+                // Rotate the momentum
+                {
```

Comment:
This part would probably require some discussion.
The idea here is that we should perform the rotation in the case when we use the `GaussianFlux` momentum distribution (because e.g. when emitting in the `r` direction, the distribution should v*Gaussian in `r`, whereas the code by default produces v*Gaussian in `x`). However, the current code performs the rotation for any momentum distribution, which is probably not desirable.

Reply:
I think this rotation should be fine for all momentum distributions. If the distribution is isotropic, then the rotation does no harm. If the distribution is not isotropic (like `GaussisnFlux`) then the rotation is correct since it gets the normal and tangential components correct. Note that the rotation would only be needed if `plasma_injector->flux_normal_axis==0`.

Reply:
OK regarding `plasma_injector->flux_normal_axis==0`: I will add this in the PR.
Regarding the rotation for any profile: what made me hesitate e.g. when users use the parser option for the momentum like here: https://github.com/ECP-WarpX/WarpX/blob/development/Examples/Tests/Langmuir/inputs_2d_multi_rz_rt#L66
In this case, we do want the `ux`, `uy` to correspond to the Cartesian component, and rotating the component might be incorrect. (Granted, the above example does not use the flux injection, but the volume injection instead).